src/gamma_inc.jl

```diff
@@ -670,7 +670,7 @@ function gamma_inc_temme_1(a::Float64, x::Float64, z::Float64, ind::Integer)
     l = x/a
     y = -a * LogExpFunctions.logmxp1(l)
     if a*eps()*eps() > 3.28e-3
-        throw(DomainError((a, x, ind_), "P(a,x) or Q(a,x) is computationally indeterminant in this case."))
+        throw(DomainError((a, x, ind), "P(a,x) or Q(a,x) is computationally indeterminant in this case."))
```

Comment:
Can you please add a test exercising this code path?

Reply:
I could, but I am not sure you want this... ;-) A couple of points:

1. The decrease in the coverage only due to the fact that I deleted a single character in the code
2. I do not see how I might have done something wrong: `ind_` is never defined in the function `gamma_inc_temme_1`, so unless you are using global variables (which I hope you are not...) it's hard to believe that variable could ever exists
3. I could not easily find any simple test on `gamma_inc`, which is the function a user would normally call
4. I did find a simple test on `gamma_inc_temme_1`: for example
````
@test_throws DomainError SpecialFunctions.gamma_inc_temme_1(1e30, 0.2, 0.3, 1)
````
but I am not sure you want to pollute the test files with obviously wrong calls to functions that are not exported.